docs/src/reference/v1beta2-exp-api.md

```diff
@@ -0,0 +1,3 @@
+# v1beta2 exp API
+
+{{ #include v1beta2-exp-api-raw.html }}
```

Comment:
when we commit this, this particular file wont exists right? Is there a need to commit this file? Or is it that when the book job runs and the files will be generated and included in the book? If yes, then we are ok

Reply:
yeah from my above comment

> I'm still not 100% sure if we want to commit the raw generated files or ignore them. I'm leaning towards committing them but I want to test how the book generation works in github actions.